hotsos/defs/scenarios/test/maas/maas_proxy_issue.yaml

```diff
@@ -0,0 +1,11 @@
+data-root:
```

Comment:
This test needs to live under hotsos/defs/tests/scenarios/maas/ otherwise it will not be run. Since it will be the first scenario test you can also delete hotsos/defs/tests/scenarios/maas/placeholder-not-a-test.yaml.

With the changes I recommended above the issue message will be:

```
raised-issues:
  MAASWarning: >-
    maas-proxy is reporting failure to forward requests (e.g. to
    api.snapcraft.io:443). You can try increasing forward_max_tries to 50 or
    setting client_persistent_connections off, server_persistent_connections off
    to alleviate the problem.
```